yui/build/moodle-atto_bsgrid-button/moodle-atto_bsgrid-button-debug.js

```diff
@@ -63,15 +63,15 @@ var col3_template =
 
 var col1x3_template =
     '<div class="atto_bsgrid container-fluid">' +
-        '<div class="row row-fluid">' +
+        '<div class="row-fluid">' +
```

Comment:
I believe these were changed prior to my fix, but someone forgot to run grunt before they merged the changes into the master branch of the original repo.